apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -225,11 +227,24 @@ where
                     // We remove the corresponding wrapper tx from the queue
                     self.storage.tx_queue.pop();
                     let mut event = Event::new_tx_event(&tx_type, height.0);
-                    if let DecryptedTx::Undecryptable(_) = inner {
-                        event["log"] =
-                            "Transaction could not be decrypted.".into();
-                        event["code"] = ErrorCodes::Undecryptable.into();
+
+                    match inner {
```

Comment:
What are these changes?

Reply:
just some logging, I think ray is going to split it